fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/serialization/JsonParserHelper.java

```diff
@@ -454,7 +454,7 @@ public static LocalDateTime convertDateTimeFrom(final String dateTimeAsString, f
         LocalDateTime eventLocalDateTime = null;
         if (StringUtils.isNotBlank(dateTimeAsString)) {
             try {
-                eventLocalDateTime = DateTimeFormat.forPattern(dateTimeFormat).withLocale(clientApplicationLocale)
+            eventLocalDateTime = DateTimeFormat.forPattern(dateTimeFormat).withLocale(clientApplicationLocale)
```

Comment:
Since no changes are made in this file, kindly remove this file from PR

Reply:
Ok